packages/gatsby/src/schema/types/built-in-types.ts

```diff
@@ -76,13 +76,15 @@ const site = `
   type Site implements Node @infer {
     buildTime: Date @dateformat
     siteMetadata: SiteSiteMetadata
+    head: Boolean
```

Comment:
Needed to include this in the default GraphQL schema so `head/index.js` could query for it even if users don't define it.

Reply:
If this is going in siteMetadata, could the key be more descriptive? "head" is a bit vague.

Reply:
It's in the root right now, but we could still make it more descriptive. Any ideas for better names?

Reply:
`enableHeadComponent` or something along those lines. Something that can be phrased as a question